dspace-api/src/main/java/org/dspace/app/util/DCInput.java

```diff
@@ -234,6 +239,9 @@ public DCInput(Map<String, String> fieldMap, Map<String, List<String>> listMap,
         if ("complex".equals(inputType)) {
             complexDefinition = complexDefinitions.getByName((fieldMap.get(DCInputsReader.COMPLEX_DEFINITION_REF)));
         }
+        if ("autocomplete".equals(inputType)) {
+            autocompleteCustom = fieldMap.get(DCInputsReader.AUTOCOMPLETE_CUSTOM);
```

Comment:
you should use setAutocompleteCustom

Reply:
Yes, but the code will be inconsistent because the `set` function is not used anywhere.